src/index.js

```diff
@@ -24,6 +27,9 @@ function splitGuide() {
 }
 
 function deletePreviouslyGeneratedFiles() {
+  if (argv.noclean) {
```

Comment:
FYI: You're probably going to have to remind me to get back to this PR after a few weeks. haha. (sorry).